sources/packages/backend/workflow/test/2024-2025/parttime-assessment/costs/parttime-assessment-costs-program-related-costs.e2e-spec.ts

```diff
@@ -0,0 +1,61 @@
+import { PROGRAM_YEAR } from "../../constants/program-year.constants";
+import {
+  ZeebeMockedClient,
+  createFakeConsolidatedPartTimeData,
+  executePartTimeAssessmentForProgramYear,
+} from "../../../test-utils";
+
+describe(`E2E Test Workflow parttime-assessment-${PROGRAM_YEAR}-program-related-costs.`, () => {
+  it(
+    "Should use minimum (offering program related costs) " +
+      "when dmn limit for books and supplies * number of offering weeks has a higher value.",
+    async () => {
```

Comment:
Minor suggestion. The test case description is great, and maybe we could change this to "Should get program related costs as offering program related costs when the DMN limit for books and supplies multiplied by offering weeks has the greater value."